pynest/examples/pynest-microcircuit/stimulus_params.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+#
+# stimulus_params.py
+#
+# This file is part of NEST.
+#
+# Copyright (C) 2004 The NEST Initiative
+#
+# NEST is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 2 of the License, or
+# (at your option) any later version.
+#
+# NEST is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NEST.  If not, see <http://www.gnu.org/licenses/>.
+
+'''
+pynest microcicuit stimulus parameters
+----------------------------------------------
+
+Stimulus parameters for the microcircuit.
+
+Hendrik Rothe, Hannah Bos, Sacha van Albada; May 2016
+'''
+
+import numpy as np
+
+stim_dict = {
+    # Turn thalamic input on or off (True or False)
+    'thalamic_input': False,
+    # Turn DC input on or off (True or False)
+    'dc_input': False,
+    # Number of thalamic neurons
+    'n_thal': 902,
+    # Amplitude of the thalamic postsynaptic potential (in mV)
+    'PSP_th': 0.15,
+    # Start of the thalamic input (in ms)
+    'th_start': 450.0,
+    # Duration of the thalamic input (in ms)
+    'th_duration': 10.0,
+    # Rate of the thalamic input
+    'th_rate': 80.0,
```

Comment:
- The thalamic rate in the SLI microcircuit is 120 1/s. What is the reason for 80 here?
- The default values for the time interval of thalamic input could be the same for the SLI and the PyNEST version.